### DISCUSSION TITLE
How to add a new button to the List view?

### DISCUSSION THREAD

**Discussion by apoorvpandey0** [crccheck/django-object-actions#131] [Q&A]
I checked that we can add buttons to individual objects, but how do we add a button to he main listview where all the objects are listed in Django Admin, I dont know if we call the some view but I hope I'm clear.

**Comment by crccheck** [comment]
I could have sworn I had a way to add a button, but my "official" response is to use an admin action instead https://docs.djangoproject.com/en/4.0/ref/contrib/admin/actions/

it's not a button but it is supported directly by Django. Buttons would be nice as a UI option but I'm guessing past me didn't look into it since the functionality was already possible to avoid technical debt
